torch/csrc/jit/passes/onnx.cpp

```diff
@@ -226,7 +226,94 @@ void BlockToONNX(
 
   // Finally, visit all nodes in the graph
   for (auto node : old_block->nodes()) {
-    if (node->kind() == prim::PythonOp) {
+    if (node->kind().is_caffe2()) {
+      auto new_node = ctx.block->appendNode(
+          ctx.block->owningGraph()->createClone(node, envFn));
+      for (size_t i = 0; i < node->outputs().size(); i++) {
+        env[node->outputs()[i]] = new_node->outputs()[i];
+      }
+      const auto& schema = node->schema();
+      size_t tensor_input_num = 0;
```

Comment:
this code might be easier to read if you were adding inputs instead of removing them. Also a short comment (or a pointer to the source of truth description) of what semantics here is would be useful - i.e. explain that all non-tensor types become arguments